core/drivers/ls_dspi.c

```diff
@@ -0,0 +1,450 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2021 NXP
+ *
+ * DSPI Driver for DSPI Controller
```

Comment:
- The first DSPI is redundant
- Maybe you could mention the meaning of DSPI here and in the commit description.